### DISCUSSION TITLE
可否添加知识汇总功能

### DISCUSSION THREAD

**Discussion by gaochuanchao** [windingwind/zotero-better-notes#444] [Ideas]
在整理论文时, 我想把所有文章描写研究背景的句子汇总到某一个可以指定的笔记里面 (不一定是主笔记), 而不是每篇文章单独创建一个笔记. 不知道这种功能能不能实现. 我感觉这种功能在写论文时很实用, 不用一篇文章一篇文章地重复查阅.

**Comment by windingwind** [comment]
可以通过笔记模板功能实现的哦

**Comment by code-thx** [comment]
> 可以通过笔记模板功能实现的哦

对于笔记模板的使用一直是个困扰，有没有大神可以出个详细的教程啊。或者开发大佬是否可以在issue中征集一下有没有朋友可以出个详细的Bnotes的使用教程。感觉上它遍的越来越强大，但是很多最初的功能现在都不知道如何使用，很怀念1.0之前的版本的很多功能。

**Reply by windingwind** [reply]
基本上参考github上的文档即可。有什么语法或者概念不清楚的可以提问
